apps/participants/serializers.py

```diff
@@ -102,6 +103,10 @@ def __init__(self, challenge_participant_team_list):
 class ChallengeParticipantTeamListSerializer(serializers.Serializer):
     """Serializer to map a challenge's participant team lists."""
     challenge_participant_team_list = ChallengeParticipantTeamSerializer(many=True)
+    time = serializers.SerializerMethodField()
+
+    def get_time(self, obj):
+        return timezone.now().replace(microsecond=0).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
@Ram81 I think `return timezone.now().replace(microsecond=0)` would work. Can you please remove `strftime`.